src/prescriptions/prescription-tab-panel.component.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from "react";
+import {
+  DataTable,
+  DataTableSkeleton,
+  Pagination,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableExpandedRow,
+  TableExpandHeader,
+  TableExpandRow,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TabPanel,
+} from "@carbon/react";
+
+import { useTranslation } from "react-i18next";
+
+import { formatDatetime, parseDate } from "@openmrs/esm-framework";
+import PrescriptionExpanded from "../components/prescription-expanded.component";
+import { usePrescriptionsTable } from "../medication-request/medication-request.resource";
+import { PrescriptionsTableRow } from "../types";
+import styles from "./prescriptions.scss";
+
+const columns = [
+  { header: "Created", key: "created" },
+  { header: "Patient name", key: "patientName" },
+  { header: "Prescriber", key: "prescriber" },
+  { header: "Drugs", key: "drugs" },
+  { header: "Last dispenser", key: "lastDispenser" },
+  { header: "Status", key: "status" },
+];
+
+interface PrescriptionTabPanelProps {
+  searchTerm: string;
+  status: string;
+}
+
+const PrescriptionTabPanel: React.FC<PrescriptionTabPanelProps> = ({
+  searchTerm,
+  status,
+}) => {
+  const { t } = useTranslation();
+  const [page, setPage] = useState(1);
+  const [pageSize, setPageSize] = useState(10);
+  const [nextOffSet, setNextOffSet] = useState(0);
+  const { prescriptionsTableRows, mutate, isError, isLoading, totalOrders } =
+    usePrescriptionsTable(pageSize, nextOffSet, searchTerm, status);
+  const encounterToPatientMap = {};
+
+  useEffect(() => {
+    if (prescriptionsTableRows?.length > 0) {
+      prescriptionsTableRows.map((order: PrescriptionsTableRow) => {
+        encounterToPatientMap[order.id] = order.patientUuid;
+      });
+    }
+  }, [prescriptionsTableRows]);
+
+  // reset back to page 1 whenever search term changes
+  useEffect(() => {
```

Comment:
This effect s new, as I noticed unpredictable behavior when you entered a search term and weren't on page 1